gallery/src/pageConfig.ts

```diff
@@ -69,7 +70,8 @@ const pageConfig: PageConfig = {
     'nx-container-helpers': NxContainerHelpersPage
   },
   'Layout Examples': {
-    'Form Layout Styles': NxFormLayoutPage
+    'Form Layout Styles': NxFormLayoutPage,
+    'Page Layout Guidelines': PageLayoutPage
```

Comment:
Should this be named "... Guidelines" if it's not in the Guildelines section?

Reply:
> Should this be named "... Guidelines" if it's not in the Guildelines section?

All the guidelines are gone here: https://github.com/sonatype/sonatype-react-shared-components/pull/28/commits/e982547a0fd7c62941a7a6975f5b823f25ee551b and here: https://github.com/sonatype/sonatype-react-shared-components/pull/28/commits/e982547a0fd7c62941a7a6975f5b823f25ee551b